src/Modules/Manage/Dnn.PersonaBar.Users/Components/Prompt/Commands/ListUsers.cs

```diff
@@ -96,13 +96,13 @@ public override ConsoleResultModel Run()
             if (!string.IsNullOrEmpty(Username))
             {
                 // do username lookup
-                var searchTerm = Username.Replace("%", "").Replace("*", "");
+                var searchTerm = Username.Trim().TrimEnd(new char[] { '%', '*' }); // Username.Replace("%", "").Replace("*", "");
```

Comment:
I think this should be changed so that leading/trailing asterisk is changed to the percent sign.